homeassistant/components/advantage_air/select.py

```diff
@@ -33,7 +37,7 @@ class AdvantageAirMyZone(AdvantageAirEntity, SelectEntity):
     _number_to_name = {0: ADVANTAGE_AIR_INACTIVE}
     _name_to_number = {ADVANTAGE_AIR_INACTIVE: 0}
 
-    def __init__(self, instance, ac_key):
+    def __init__(self, instance: advantage_air, ac_key: int) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3